instabug/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "37d9bc39-888f-4bec-b8c5-3c137cf88f84",
+  "app_id": "instabug",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and track your mobile app health and performance",
+    "title": "Instabug",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Use the Instabug widget to monitor the overall performance of your app.",
+        "image_url": "images/instabug-datadog-widget.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Instabug's dashboard gives teams a succinct overview of the apps performance and potential failures or issues occurring in the app.",
+        "image_url": "images/instabug-app-overview.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "With each feedback or bug your users submit, we automatically capture all the details you need to know why users are facing issues with your code.",
+        "image_url": "images/instabug-bugs.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Prioritize resolution by the most impactful crashes to your users.",
+        "image_url": "images/instabug-crashes-list.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Understand how each version of your app release is performing and overall adoption.",
+        "image_url": "images/instabug-releases-page.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Aggregate data across device and version to better understand where issues are occurring.",
+        "image_url": "images/instabug-crash-details.png"
+      }
+    ],
+    "classifier_tags": [
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Alerting",
+      "Category::Monitoring",
+      "Category::Issue Tracking"
+    ]
+  },
+  "assets": {},
```

Comment:
It looks like the mandatory fields under assets are missing. Please include the field [here](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file)

Reply:
@emarsha94 I can see that all the fields for assets was optional and we don't use any of them that's why I removed them
![image](https://user-images.githubusercontent.com/40190772/215846568-f103502e-4ed0-47e6-b2b7-978eb79da2c5.png)

Reply:
@emarsha94 Could u please specify what are the fields we need here

Reply:
Hi @AbdallahHemdan All of the fields under assets in this [article](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file) listed as mandatory need to be added back. It may be easier to re-add everything that was deleted under assets and we can remove what isn't needed from there. For example, events, metrics, service checks, etc. need to be re-added.